ext/standard/basic_functions.stub.php

```diff
@@ -361,7 +361,7 @@ function is_uploaded_file(string $path): bool {}
 
 function move_uploaded_file(string $path, string $new_path): bool {}
 
-function parse_ini_file(string $filename, bool $process_sections = false, int $scanner_mode = INI_SCANNER_NORMAL): array {}
+function parse_ini_file(string $filename, bool $process_sections = false, int $scanner_mode = INI_SCANNER_NORMAL): array|false {}
```

Comment:
Isn't this change the wrong way around?

Reply:
I believe it's not, because as far as I saw `parse_ini_file()` can also return with `false`: https://github.com/php/php-src/blob/2adb1bba17fccdded38b21d4ef84ca8f24747ec7/ext/standard/basic_functions.c#L4236